hang_man.py

```diff
@@ -0,0 +1,148 @@
+import os
+import sys
+import copy
+import string
+import random
+import argparse
+import urllib.request
+
+
+class Hangman:
+
+    def __init__(self, min_limit):  # Creating init method that takes in max word limit as perimeter
+        self.min_limit = min_limit  # Minimum word limit taken from user
+        self.word_list = []  # A list that stores all the word from local file or web
+        self.right_list = []  # A list that stores the right word
+        self.guess_letter = str  # User input of their guessed character
+
+    def access_doc(self):
+        file_loc = './word_data/words.txt'
+        file_dir = './word_data/'
+        url = 'http://norvig.com/ngrams/sowpods.txt'  # Create a variable that stores for URL for text file
+        file_exist = os.path.exists(file_loc)  # Boolean value to check if text file exist first
+        if not file_exist:  # If text file does not exist following happens:
+            folder_exist = os.path.isdir(file_dir)  # Checks to see if directory exists
+            if not folder_exist:  # If directory does not exist following happens:
```

Comment:
i think i meant to put the comment above this at this point...my bad. not going to change it.